um_canonicalize/tags.yaml

```diff
@@ -79,9 +84,28 @@ categories:
     - INS
     - COM
     - VOC
+    - AT+ESS
+    - AT+ESS+ALL
+    - IN+ESS
+    - IN+ALL
+    - IN+ABL
+    - AT+ALL
+    - AT+ABL
+    - ON+ESS
+    - ON+ALL
+    - ON+ABL
+    - VOC+GEN
+    - NOM+ACC
+    - non{NOM}
+    - DAT+GEN
```

Comment:
Where does this particular combination come from? I haven't seen this combination before.

Reply:
Romanian (and no others). It's proper.

Reply:
Well, let me revise that remark. It should really be `{DAT/GEN}`, but there is a slot that is genitive-like and dative-like in the language.